features/geometrisch-zoeken.feature

```diff
@@ -0,0 +1,700 @@
+
+    Deze feature beschrijft geen scenario's waarbij objecten een geometrie van het type multi polygoon bevatten.
+    Het enige object type waar dit voor geldt is een woonplaats, echter wordt het zoeken van woonplaatsen met locatie, bbox of polygoon niet ondersteund.
+
+    Het zoeken met een multipolygoon wordt bij adressen ondersteund.
+    Deze scenario's zijn niet apart beschreven, omdat het zoeken met een multipolygoon overeenkomt met het meerdere malen zoeken met een polygoon waarbij het resultaat wordt ontdaan van evt. dubbelingen.
+
+    In de hierna volgende voorbeelden is een vereenvoudigde representatie gebruikt voor coördinaten.
+    
+    Voor een visualisatie van de scenario's zoals hieronder beschreven zie:
+    - locatie voorbeelden: /images//locatie-voorbeelden.png
+      De zwarte punten zijn de locaties waarmee wordt gezocht. 
+      De overige figuren zijn de geometrieën van objecten.
+
+    - bounding box voorbeelden: /images/bbox-voorbeelden.png
+      De zwarte onderbroken lijnen geven de bbox aan waarmee wordt gezocht.
+      De zwarte punt is de bounding box waarbij het punt linksonder en rechtsboven gelijk zijn en waarmee wordt gezocht. 
+      De overige figuren zijn de geometrieën van objecten.
+
+    - polygoon voorbeelden: /images/locatie-voorbeelden.png
+      De zwarte onderbroken lijnen geven de bbox aan waarmee wordt gezocht. 
+      De overige figuren zijn de geometrieën van objecten.
+
+  Achtergrond:
+
+    Gegeven de volgende verzameling van resources
+
+    | identificatie | geometrie                                 | geometrie type |
+    |---------------|-------------------------------------------|----------------|
+    | 0001          | [[0,0],[10,0],[10,10],[0,10],[0,0]]       | polygoon       |
+    | 0002          | [[12,0],[22,0],[22,10],[12,10],[12,0]]    | polygoon       |
+    | 0003          | [[12,12],[22,12],[22,22],[12,22],[12,12]] | polygoon       |
+    | 0004          | [[0,8],[10,8],[10,18],[0,18],[0,8]]       | polygoon       |
+    | 0005          | [7,20]                                    | punt           |
+    | 0006          | [17,17]                                   | punt           |
+    | 0007          | [10,23]                                   | punt           |
+    | 0008          | [13,21]                                   | punt           |
+    
+    
+  Scenario: GET met locatie, Content-Crs (optioneel) en Accept-Crs (optioneel) één resource met geometrie type polygoon, locatie valt binnen polygoon
+  
+    Als er met de volgende parameters wordt gezocht:
+    | locatie | Content-Crs | Accept-Crs |
+    |---------|-------------|------------|
+    | [5,5]   | -           | -          |
+    | [5,5]   | EPSG:28992  | -          |
+    | [5,5]   | -           | EPSG:28992 |
+    | [5,5]   | EPSG:28992  | EPSG:28992 |
+    En de provider ondersteunt het CRS in Content-Crs
```

Comment:
onder de Als vallen acties van de gebruiker. Precondities, zoals wat de provider ondersteunt, horen bij Gegeven. Maar in dit geval is die ook daar niet nodig, want de specificaties schrijven al voor dat de provider EPSG:28992 ondersteunt.